api/hearings/models/hearings.enum.ts

```diff
@@ -48,7 +48,7 @@ export enum EXUIDisplayStatusEnum {
   FAILURE = 'FAILURE',
   CANCELLATION_REQUESTED = 'CANCELLATION REQUESTED',
   VACATED = 'VACATED',
-  AWAITING_ACTUALS = 'AWAITING ACTUALS',
+  AWAITING_ACTUALS = 'AWAITING HEARING ACTUALS',
```

Comment:
As per latestest prototype the label is now 'AWAITING HEARING DETAILS'

Reply:
Yes, that's what this label is

Reply:
scope creep, originally was hearing actuals